OIOUBL/src/Management/COD13646.Management.al

```diff
@@ -20,7 +20,7 @@ codeunit 13646 "OIOUBL-Management"
 
         OIOUBLFileEvents.FileCreated(SourceFile);
 
-        if FileManagement.CanRunDotNetOnClient() then begin
+        if not ExportFileFromEvent(SourceFile) and FileManagement.CanRunDotNetOnClient() then begin
```

Comment:
@MarcHansenMicrosoft Can you help me out here, I'm out of ideas on how I can test that the file is not downloaded, when I return IsExported=true from the ExportFileFromEvent function.

Reply:
@RasmusTidselbak sorry for the slow response time.

First I do not think you should test for what the program should not do.

And I also think you should restructure the code to make it more clear that first you call the publisher to handle the save if it, then you check if it can save using com and last you just upload to the client.